.changeset/@theguild_eslint-config-154-dependencies.md

```diff
@@ -0,0 +1,5 @@
+---
+"@theguild/eslint-config": patch
+---
+dependencies updates:
+  - Removed dependency [`eslint-plugin-simple-import-sort@^8.0.0` ↗︎](https://www.npmjs.com/package/eslint-plugin-simple-import-sort/v/8.0.0) (from `dependencies`)
```

Comment:
can `prettier-plugin-sort-imports` sort exports too? if not let's keep this plugin but for only for exports

Reply:
It cannot. But I wouldn't sort the exports through eslint because we'd be reverting the annoyance I am trying to remove here. 😅

Why do we want to sort exports, any concrete reason except aesthetics? 

The whole idea is that format-on-save does the "beautification", and that should be handled by prettier; eslint, on the other hand, should only enforce rules on semantics, not visuals.

Reply:
Ok, I’m ok with your opinion Denis, let’s remove